.github/workflows/mysql_performance.yml

```diff
@@ -38,17 +38,14 @@ jobs:
           echo "TEMP_AWS_SESSION_TOKEN=${creds[2]}" >> $GITHUB_ENV
       - name: 'Run performance tests (OpenJDK)'
         run: |
-          ./gradlew --no-parallel --no-daemon test-performance-aurora-mysql
+          ./gradlew --no-parallel --no-daemon test-all-environments -Ptest-no-docker=true -Ptest-no-iam=true -Ptest-no-hikari=true -Ptest-no-secrets-manager=true -Ptest-no-graalvm=true -Ptest-no-mariadb-driver=true -Ptest-no-mariadb-engine=true -Ptest-no-pg-engine=true -Ptest-no-pg-driver=true
```

Comment:
On second thought, would it be more intuitive to use if we register a new task similar to "test-all-aurora-performance" for both mysql or postgres?